securedrop/db.py

```diff
@@ -274,20 +287,31 @@ def _scrypt_hash(self, password, salt, params=None):
         return scrypt.hash(str(password), salt, **params)
 
     MAX_PASSWORD_LEN = 128
-    MIN_PASSWORD_LEN = 12
+    MIN_PASSWORD_LEN = 14
```

Comment:
Because the minimum of 7 "diceware-like" is 14. `'a ' * 7`. Well, actually 13, but whatever. It's just for testing.